metaci/plan/models.py

```diff
@@ -43,6 +43,7 @@
     ("medium", "medium priority"),
     ("high", "high priority"),
     ("robot", "robot tests"),
+    ("perftests", "performance tests"),
```

Comment:
I think this queue should be named based on how it operates (i.e. oneoff) rather than a particular use case. It'll be tied to performance tests by virtue of being the queue selected for performance test plans.